yt/visualization/base_plot_types.py

```diff
@@ -489,7 +489,7 @@ def _toggle_axes(self, choice, draw_frame=None):
             # last checked with Matplotlib 3.5
             warnings.warn(
                 f"Previously set background color {self.colorbar_handler.background_color} "
-                "has no effect. Pass `draw_axis=True` if you wish to preserve background color.",
+                "has no effect. Pass `draw_axes=True` if you wish to preserve background color.",
```

Comment:
given the context I think the intention was to adversative `draw_frame`
```suggestion
                "has no effect. Pass `draw_fram=True` if you wish to preserve background color.",
```